rocky/tools/forms/findings.py

```diff
@@ -0,0 +1,77 @@
+from django import forms
+from django.utils.translation import gettext_lazy as _
+
+from octopoes.models.ooi.findings import (
+    ADRFindingType,
+    CAPECFindingType,
+    CVEFindingType,
+    CWEFindingType,
+    KATFindingType,
+    RetireJSFindingType,
+    RiskLevelSeverity,
+    SnykFindingType,
+)
+from tools.forms.base import BaseRockyForm
+
+FINDING_TYPES_LIST = [
+    ADRFindingType,
+    CVEFindingType,
+    CWEFindingType,
+    CAPECFindingType,
+    RetireJSFindingType,
+    SnykFindingType,
+    KATFindingType,
+]
+FINDING_TYPES_CHOICES = (
+    (str(finding_type.__name__), str(finding_type.__name__)) for finding_type in FINDING_TYPES_LIST
+)
+
+FINDINGS_SEVERITIES_CHOICES = (
+    (str(severity.name).lower(), str(severity.value).lower()) for severity in RiskLevelSeverity
+)
+
+MUTED_FINDINGS_CHOICES = (
+    ("show", _("Show Muted Findings")),
+    ("exclude", _("Exclude Muted Findings")),
+)
+
+
+class FindingSeverityMultiSelectForm(forms.Form):
+    severity = forms.MultipleChoiceField(
+        label=_("Filter by severity"),
+        required=False,
+        choices=FINDINGS_SEVERITIES_CHOICES,
+        widget=forms.CheckboxSelectMultiple,
+    )
+
+
+class FindingRiskScoreNumberForm(forms.Form):
```

Comment:
If the fields aren't required but do have boundaries, then they should have a default value right?

Reply:
So to clarify this more:

No filtering for risk scores (no values) = show all findings
![image](https://github.com/minvws/nl-kat-coordination/assets/30832687/c076e441-7681-4343-8849-7367aa71d418)

one value inserted, in this case min risk score, filters everything from 3.5 up till max 10
![image](https://github.com/minvws/nl-kat-coordination/assets/30832687/b4300ff7-5337-424f-8314-26e4cd6d80b2)

max value inserted, filter till the max value, from 0 up
![image](https://github.com/minvws/nl-kat-coordination/assets/30832687/b114a9b5-b52e-4a24-8e8e-fac73d9f889e)

Filter all in between values:
![image](https://github.com/minvws/nl-kat-coordination/assets/30832687/313fbf4f-de9d-4230-98d2-98aa5af4cead)